### PR TITLE
Add telco minute lines to home page

### PR DESCRIPTION
**Pull request by BigBlueHat**
This highlights that we are actually doing stuff 😉 

json-ld-wg-with-telco-minutes-on-home-page

### CODE REVIEW

File: _layouts/home.html [hunk 1]
```diff
@@ -37,6 +45,14 @@ <h1 style="border-bottom: 1px solid #ccc">
             <section id="mission">
                 {{ content }}
             </section>
+            <section id="minutes">
+                <h2>Telco Minutes</h2>
+                <ul class="inline">
+                {% for item in site.minutes reversed %}
```
Comment (davidlehn) [inline _layouts/home.html]: Probably should limit this to a handful of recent minutes and a "more..." link to the minutes page.  Would be a bit cluttered after we've had many meetings.

Comment (BigBlueHat) [reply _layouts/home.html]: 👍 fixed in 7ccf60f

**Comment by gkellogg** [comment]
Links to minutes on the home page bring up a 404.

**Comment by iherman** [comment]
@gkellogg  it works for me:-(

**Comment by gkellogg** [comment]
Hmm, https://w3c.github.io/json-ld-wg/ shows the minute links, which resolve, say, to 
https://w3c.github.io/2018/json-ld-wg/Meetings/Minutes/2018/2018-08-03-json-ld, for which I get a 404. Actually, when accessed from w3c.github.io, many links don’t work.

When accessed from https://w3.org/2018/json-ld-wg things seem to work okay. It seems that either links should go explicitly to w3.org, or we need something to direct people there from the github site.

**Comment by iherman** [comment]
Ah. I have to forward this issue to @BigBlueHat then, because I did not dive into all the things that he did. 

@BigBlueHat in your script you made use of a 'base' setting in the jekyll config file and you use that in the script that generates the addresses for the minutes. That mechanism may go wrong... But hat also generates a wrong URL, see the URL cited by @gkellogg...

Yes, indeed, it is correct: in many cases one has to explicitly use the W3C URL to get everything working, and the github.io should not be used explicitly. I _think_ the main problem is the interpretation of the 'base': when coming from W3C then, well, it is W3C, whereas when coming from .io then it is the .io level and that does not work.

(E.g., using a relative URL for CSS files does not work, it has to be an absolute URL using the W3C URL...)

**Comment by BigBlueHat** [comment]
@gkellogg yeah...this is where the whole URL thing falls over...

In short...this:
> Yes, indeed, it is correct: in many cases one has to explicitly use the W3C URL to get everything working, and the github.io should not be used explicitly. I think the main problem is the interpretation of the 'base': when coming from W3C then, well, it is W3C, whereas when coming from .io then it is the .io level and that does not work.

Expanding on @iherman's thoughts on this topic...

This same content is used in 3 distinct locations:
 - https://www.w3.org/2018/json-ld-wg/ (what one would consider canonical)
 - https://w3c.github.io/json-ld-wg/ (GitHub's auto-magic URL)
 - `http://localhost:4000/2018/json-ld-wg/` (when testing locally)

The GitHub one is the "odd one out" because we can't adjust it to match the W3C's URL publication style (i.e. beginning with the date of the creation of the group, then the group name, etc).

Sadly, I've not found a way to contextually change these to match their locations...which means the GitHub one looses... I'd though briefly (and the comments in my commit messages show my misconceptions), that GitHub was doing some automatic `baseurl` settings when hosted on their server...but alas.

If there's a way to make URL containing content survive being hosted in 3 different prefixed locations...I've not found it... Let me know if y'all do. 😃

**Comment by davidlehn** [comment]
For the minutes links can you just use relative links?  Looks like maybe you could take the leading '/Meetings/Minutes/' part off the _config.yml permalink, change the home page site.baseurl to 'Meetings/Minutes/', and drop the site.baseurl part in the minutes index.md.  Or am I just naive about this? ;-)

**Comment by iherman** [comment]
@davidlehn yes, this may work with the minutes. 

The problem does have an effect on the site structure overall. The problem I have ran into is the template files that have to use absolute and W3C URL-s for, eg, CSS files, even if they are in the repository (otherwise they do not work on the full hierarchy). Many times have I burned my fingers on this, when testing some CSS features on my own machine (running a local jekyll server) and not understanding why changes did not have any effect.

IMHO we should try to get rid of the 'base' statement in the jekyll config file. I am not sure it helps, and it can lead to confusion (it did for me when I tested some late changes, because my local bookmarks on localhost went all wrong:-(

**Comment by BigBlueHat** [comment]
The `site.baseurl` is set in `_config.yml` and is for the entire site. It sadly (afaik) can't be conditionally configured, so [currently it's `/2018/json-ld-wg`](https://github.com/w3c/json-ld-wg/blob/733f45d73098da6328057a9df621eb9b651dbdfe/_config.yml#L1) to match the W3C's publication path.

The tricky bit isn't content files referencing other content files (assuming they don't move around...), but rather layouts and includes which are used in multiple places.

So...
> maybe you could take the leading '/Meetings/Minutes/' part off the _config.yml permalink
...sadly won't work because the `permalink` needs to include that so the files are generated in the same place as ever (i.e. so we don't 404 all the old links to the minutes).

> change the home page site.baseurl to 'Meetings/Minutes/'
The `site.baseurl` is global, so we can't (or at least shouldn't) conditionally set that. The `site.baseurl` is also meant to be used for dependency CSS (etc) pathing and such, so it needs to be consistent (even if it's blank/empty).

> drop the site.baseurl part in the minutes index.md
This is possible in `Meetings/Minutes/index.md`, but it wouldn't work for scenarios using templates or includes as those aren't always used in the same hierarchical space.

I'm still digging for a solution that gets all 3 of these scenarios in sync, but for now "breaking" the non-official github.io links seems the most pragmatic cost exchange...

Feel free to file another issue if you think it's worth discussions further and/or solving together. 😁

**Comment by BigBlueHat** [comment]
This locked issue on Jekyll's issues shows there's a misunderstanding of relativity: https://github.com/jekyll/jekyll/issues/6360

And that...sadly...is the root (heh) cause of these issues. 😜

**Pull request merged**